### ISSUE TITLE
Hot Reload Fails / Requires Page Refresh

### ISSUE THREAD

**Issue by ATSiem** [owen2345/rails-hotreload#1]
Coming from a Flutter background, Hot Reload is the best! Excited to get this working for Rails 7, and share the dev flow with Rails Community.

I've got rails-hotreload installed per the Github guide. My page demonstrates the 'hot reload' toggle (on), and the rails_hotreload process detects the changes, and reloads, but I'm not seeing them live on the page *until* I refresh the browser.

Any tips on how to test / resolve?

![image](https://user-images.githubusercontent.com/1068543/154198254-2e7e9bb0-d177-4065-9837-4ab0adc080b8.png)

![image](https://user-images.githubusercontent.com/1068543/154198283-96fbeedf-dafb-45f0-8c10-d1a447970562.png)

![image](https://user-images.githubusercontent.com/1068543/154198310-827daf8d-421f-45bb-9cfc-c52e5b1910be.png)

**Comment by owen2345** [comment]
@adamsiem I think your problem is with Turbo dependency which is in charge of streaming the changes from the server.
Try this (or you can manually configure your turbo settings into redis):
<img width="962" alt="Screen Shot 2022-02-16 at 5 56 24 AM" src="https://user-images.githubusercontent.com/1148415/154240370-2c41c776-c3e5-4c8d-be5c-266f051f91ff.png">

I am still thinking to use action-cable instead of Turbo to notify server changes to the frontend.

**Comment by ATSiem** [comment]
Awesome. It works now. Redis was not installed/configured.

I ran `./bin/rails turbo:install:redis`

But, it still did not work, despite updating bundle.

Then, I i[nstalled Redis and started the Redis service locally with brew](https://gist.github.com/tomysmile/1b8a321e7c58499ef9f9441b2faa0aa8).

`brew update`
`brew install redis`
`brew services start redis`

I confirmed I have a server running with the following commands:

`redis-cli ping` >> responded PONG
`ps aux | grep redis` >> responded with something that looks like this:
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/1068543/154290637-8d99a704-0812-4418-a1be-98a02d67c116.png">

Now, each time I make a change to one of my views, and save, I instantly see the change reflected on my external monitor display my local dev copy. :) I can keep my hands and keys in the dev tools.

I tested and it works for both my HTML/Forms, as well as my Tailwind CSS builds.
